### PR TITLE
Akka 2.5.27

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: blog/_posts/2019-12-10-akka-2.5.27-released.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+* Avoid too early shutdown of Sharding coordinator, which could result in wrong state when graceful leaving [#28093](https://github.com/akka/akka/issues/28093). Thanks to [@retriku](https://github.com/retriku).
+* Backport serialization support for StreamRefs [#28215](https://github.com/akka/akka/issues/28215)
+* Backport improved ByteString.toArray performance under Scala 2.13 [#28114](https://github.com/akka/akka/issues/28114).
+
+A total of 8 issues were closed since 2.5.26. The complete list can be found on the [2.5.27 milestone](https://github.com/akka/akka/milestone/153?closed=1) on github.
+
+## Credits
+
+For this release we had the help of 4 committers – thank you all very much!
```
Comment (raboof) [inline blog/_posts/2019-12-10-akka-2.5.27-released.md]: maybe 'all' is a bit much this time ;)

```suggestion
For this release we had the help of 4 committers – thank you very much!
```

File: blog/_posts/2019-12-10-akka-2.5.27-released.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Dear hakkers,
+
+We are pleased to announce a new patch release of Akka 2.5. The release contains a set of smaller fixes.
+
+Notable changes:
+
+* Avoid too early shutdown of Sharding coordinator, which could result in wrong state when graceful leaving [#28093](https://github.com/akka/akka/issues/28093). Thanks to [@retriku](https://github.com/retriku).
+* Backport serialization support for StreamRefs [#28215](https://github.com/akka/akka/issues/28215)
+* Backport improved ByteString.toArray performance under Scala 2.13 [#28114](https://github.com/akka/akka/issues/28114).
```
Comment (raboof) [inline blog/_posts/2019-12-10-akka-2.5.27-released.md]: perhaps also mention that the backport of https://github.com/akka/akka/pull/28311 means log markers can be used in libraries that keep support for both akka 2.5 and 2.6?

Comment (patriknw) [reply blog/_posts/2019-12-10-akka-2.5.27-released.md]: I think that is mostly interesting for us (and not users) so far.

File: blog/_posts/2019-12-10-akka-2.5.27-released.md (hunk 1 above)
Comment (patriknw) [reply blog/_posts/2019-12-10-akka-2.5.27-released.md]: yeah, 4 is not that many (copy of standard text)

**Pull request merged**